File: JavaScript/8js 3/sixToast.js
/*
|--------------------------------------------------------------------------
| BASIC: Making Six Toast.
|--------------------------------------------------------------------------
|
*/

function sixToast(num) {
   return Math.abs(num - 6);
}


const sixToast = (_) => (_ > 6 ? _ - 6 : 6 - _);


function sixToast(num) {
   return Math.abs(6 - num);
}


console.log(sixToast(6), 0);
console.log(sixToast(17), 11);
console.log(sixToast(3), 3);

File: Python/8py 3/six_toast.py
"""
|--------------------------------------------------------------------------
| BASIC: Making Six Toast.
|--------------------------------------------------------------------------
|
"""

def six_toast(num):
   return abs(num - 6)


six_toast = lambda _: _-6 if _>6 else 6-_


def six_toast(num):
   return abs(6 - num)


print(six_toast(15), 9)
print(six_toast(6), 0)
print(six_toast(3), 3)